regular/talks.html

```diff
@@ -30,7 +30,7 @@ <h3><em>How Robot Swarms Can Help Solve Hard Problems</em> by C Wringe</h3>
 In this talk, I will go over what a robot swarm is, and the sorts of problems they can solve. I will show what it means for a robot swarm to be scalable, flexible, and robust. Finally, I will go into some of the challenges faced by the field, and areas of current and future work.
 </p>
 <h3><em>An Introduction to Constraint Programming</em> by Jacob Allen</h3>
-<b>March 18th (Spring Week 10)</b>
+<b>March 18th (Spring Week 11)</b>
```

Comment:
You'll want to change the date here too no?
```suggestion
<b>March 25th (Spring Week 11)</b>
```